src/shared/components/react-live/live-editor.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+import { LiveProvider, Editor } from 'react-live'
+
+export default class LiveEditor extends Component {
+  static contextTypes = LiveProvider.childContextTypes
+
+  constructor (props, context) {
+    super(props, context)
+    this.code = this.context.live.code
+  }
+  run () {
+    this.context.live.onChange(this.code)
+  }
+
+  render () {
+    return (
+      <Editor
+        {...this.props}
+        code={this.context.live.code}
+        onChange={code => {
```

Comment:
Use `Parenthesis` in the arrow functions, and be careful with the spaces when using `{ ... }`